xt/41-coaload.t

```diff
@@ -1,6 +1,7 @@
 use Test::More;
 use strict;
 use warnings;
+use LedgerSMB::Sysconfig;
```

Comment:
Hmm. I'm not sure I understand the remark that we assume that the LSMB machine is local. We documented as much in https://github.com/ledgersmb/LedgerSMB/tree/master/t#environment-variables : we expect the tests to be passed the database configuration through environment variables. If you want a non-local database, simply set PGHOST and PGPORT.